<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fing_key</class>
 <widget class="QDialog" name="fing_key">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>1640</x>
     <y>940</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>50</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前已有关键词：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>120</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>490</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查找具有关键字的节点</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>650</y>
     <width>511</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>590</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>查找结果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>360</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入要查找的关键词：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>380</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>return_btn</sender>
   <signal>clicked()</signal>
   <receiver>fing_key</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1719</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
